modules/test/playwright/tests/locked-items-web/locked-items-web.spec.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {productMenuPageTest} from '../../fixtures/productMenuPageTest';
+import {lockedItemsPagesTest} from './fixtures/lockedItemsPagesTest';
+
+const test = mergeTests(
+	loginTest,
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPD-11003': true,
+		'LPS-180328': true,
+	}),
+	lockedItemsPagesTest
+);
+
+const testFeatureFlagDisabled = mergeTests(
+	loginTest,
+	apiHelpersTest,
+	featureFlagsTest({
+		'LPD-11003': false,
```

Comment:
I expected to include here `'LPS-180328':  false,` too.

Reply:
If `LPS-180328=true` and `LPD-11003=false` another page called *Locked Pages* is shown, which Locked Items module do not need to test.

Reply:
Why do you expect more than one FF disabled? The name says `testFeatureFlagDisabled` (singular) not `testFeatureFlagsDisabled` (plural). And this module only has one directly related FF.

Reply:
`testFeatureFlagDisabled` has been removed